Sources/tart/Commands/Run.swift

```diff
@@ -15,10 +15,13 @@ struct Run: AsyncParsableCommand {
   var name: String
 
   @Flag(help: ArgumentHelp(
-          "Don't open a UI window.",
+          "Don't open an UI window.",
           discussion: "Useful for integrating Tart VMs into other tools.\nUse `tart ip` in order to get an IP for SSHing or VNCing into the VM.")) 
   var noGraphics: Bool = false
 
+  @Flag(help: "Force open an UI window, even when VNC is enabled.")
```

Comment:
```suggestion
  @Flag(help: "Force open a UI window, even when VNC is enabled.")
```
Same as other comment.